sdks/go/test/integration/integration.go

```diff
@@ -69,6 +69,8 @@ var directFilters = []string{
 var portableFilters = []string{
 	// The portable runner does not support the TestStream primitive
 	"TestTestStream.*",
+	// The trigger tests uses TestStream
+	"TestTrigger.*",
```

Comment:
You'll also need to add this to the samza, spark, direct, and dataflow tests. Basically anywhere TestStream is being filteed out right now.